src/main/java/uk/gov/ons/census/casesvc/service/CCSPropertyListedService.java

```diff
@@ -47,6 +59,8 @@ public void processCCSPropertyListed(ResponseManagementEvent ccsPropertyListedEv
         ccsPropertyListedEvent.getEvent(),
         convertObjectToJson(ccsProperty));
 
-    ccsToFieldService.convertAndSendCCSToField(caze);
+    if (ccsProperty.getUac() == null) {
+      ccsToFieldService.convertAndSendCCSToField(caze);
```

Comment:
no need to check for null if moved into "if (ccsProperty.getUac() == null) {" condition above